src/igr-tests/meson.build

```diff
@@ -0,0 +1,111 @@
+# Included from top-level meson.build
+
+## Integration test's Defines
+igr_tests_env = [
+     'IS_MESON=true',
+     'APPS_PATH=' + APPS_PATH
+]
+
+## Prepare integration tests
+run_command([
+    find_program('cp'),
+    '-r',
+    './',
+    meson.current_build_dir()
+    ],
+    check : true
+)
+
+## Outputs
+# Integration tests
+basic_test_script = find_program('@0@/basic/run-test.sh'.format(meson.current_build_dir()))
```

Comment:
No need for `find_program` in all these cases, I think

Reply:
Seems like without `find_program`; they don't work!

Reply:
you can use `files` instead, there is no need to search when you already have the full path

Reply:
eg
```
basic_test_script = files('@0@/basic/run-test.sh'.format(meson.current_build_dir()))
```

Reply:
thanks. With `files` i can use `meson.current_build_dir()` directly without use `format()`!